app/controllers/questions_controller.rb

```diff
@@ -56,6 +57,14 @@ def set_question
       @question = Question.find(params[:id])
     end
 
+    def set_own_question
+      if @question.created_by?(current_user)
+        @question = current_user.questions.find(params[:id])
+      else
+        redirect_to questions_url, alert: 'Selected question is not editable.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.